proto/injective/evm/v1/events.proto

```
@@ -1,6 +1,8 @@
 syntax = "proto3";
 package injective.evm.v1;
 
+import "injective/evm/v1/log.proto";
```

Buf (1.72.0)

[error] imported file does not exist

(COMPILE)